tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/php
+<?php declare(strict_types=1);
+
+/**
+ * This script will upgrade the LORIS database to a specified release version.
+ * Usage: php loris_db_upgrade.php -D 
+ * Errorlog: log/*.log
+ * PHP Version 7
+ *
+ * @category Install
+ * @package  Loris
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris/
+ * @package core
+ */
+require_once __DIR__ . "/generic_includes.php";
+
+// check the running path if not in tools, it will quit
+if (basename(__DIR__) !== 'tools') {
+    exit("This script only runs in the tools directory!");
+}
+
+//get sql file path
+$path    = __DIR__ . "/../SQL/Release_patches";
+
+// file names from the Release_patches
+$files = array_diff(scandir($path), array('.', '..'));
+
+echo "This script will upgrade the LORIS database to the latest release version.\n";
+
+if (!file_exists('../project/config.xml')) {
+    exit("config.xml file doesn't exist. Aborting update.");
```

Comment:
```suggestion
    print("config.xml file doesn't exist. Aborting update.");
    exit(1);
```
This can help in future automation